frontend/app/styles/feedback.less

```diff
@@ -8,15 +36,22 @@
 }
 
 .feedback-container {
-  text-align: center;
+  background-color:  white;
```

Comment:
please use less variable for background-color in the .feedback-container, ngdialog.ngdialog-theme-default .ngdialog-content